soh/soh/Enhancements/randomizer/randomizer_item_tracker.cpp

```diff
@@ -283,6 +283,19 @@ bool IsValidSaveFile() {
     return validSave;
 }
 
+bool HasSong(ItemTrackerItem item) {
+    uint32_t bitMask = 1 << item.id;
+    return (bitMask & gSaveContext.inventory.questItems) != 0;
```

Comment:
i think the `!= 0` here isn't needed. this could probably just be

```suggestion
    return (1 << item.id) & gSaveContext.inventory.questItems;
```